### DISCUSSION TITLE
MAOIs + Opioids, Tramadol

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#88] [Drug Interactions]
Definitely unsafe to combine MAOI with tramadol (Gillman, 2005).

I would move to reword the current explanation for MAOIs + opioids to: While this combination is relatively low risk, there are certain opioids, such as methadone, that can increase risk of serotonin toxicity when taken in combination with MAOIs.

Gillman, P. K. (2005). Monoamine oxidase inhibitors, opioid analgesics and serotonin toxicity. BJA: British Journal of Anaesthesia, 95(4), 434–441. https://doi.org/10.1093/bja/aei210

**Comment by utaninja** [comment]
Tramadol is in its category, do you mind clarifying which this is for?

This is for opioids and MAOIs.
Opioids + MAOIs =  Caution
Coadministration of monoamine oxidase inhibitors (MAOIs) with certain opioids has been associated with rare reports of severe and fatal adverse reactions. There appear to be two types of interaction, an excitatory and a depressive one. Symptoms of the excitatory reaction may include agitation, headache, diaphoresis, hyperpyrexia, flushing, shivering, myoclonus, rigidity, tremor, diarrhea, hypertension, tachycardia, seizures, and coma. Death has occurred in some cases.

For Tramadol + MAOIs it equals dangerous for the chart. No description.

Are you saying to remove the other comments for the interaction as well?

**Reply by zelixir25** [reply]
no i've just been verifying and finding sources for existing combos so we can back up the claims we're making and not just dumping links into a unreadable wiki page

**Reply by zelixir25** [reply]
also yes i would update the wording to clarify that this applies mostly to certain opioids. i don't know that we need to list symptoms of serotonin toxicity in this combo unless we're also going to list them for other combos were ST/SS is a risk. maybe we can instead just link to a page about SS/ST for people to find out more about that?

**Reply by zelixir25** [reply]
also sorry if it wasn't clear enough but tramadol + MAOI = unsafe and MAOI + opioids = caution with just a shorter explanation without a description of ST

**Reply by utaninja** [reply]
Okay so just to clarify, are you suggesting this?

Tramadol + MAOI = unsafe | Which is saying to change the current dangerous rating to unsafe rating
You are suggesting to add this study as a source: Gillman, P. K. (2005). Monoamine oxidase inhibitors, opioid analgesics and serotonin toxicity. BJA: British Journal of Anaesthesia, 95(4), 434–441. https://doi.org/10.1093/bja/aei210

Opioids + MAOI = caution | Which is suggesting to keep the same rating
You are suggesting to add this study as a source: Gillman, P. K. (2005). Monoamine oxidase inhibitors, opioid analgesics and serotonin toxicity. BJA: British Journal of Anaesthesia, 95(4), 434–441. https://doi.org/10.1093/bja/aei210
You are suggesting to change this combination message to- While this combination is relatively low risk, there are certain opioids, such as methadone, that can increase risk of serotonin toxicity when taken in combination with MAOIs.

**Reply by utaninja** [reply]
Here's what I'm thinking-

My suggestion would be:

Tramadol + MAOI = Dangerous
Increased risk of seizures and risk of serotonin syndrome.

Gillman, P. K. (2005). Monoamine oxidase inhibitors, opioid analgesics and serotonin toxicity. BJA: British Journal of Anaesthesia, 95(4), 434–441. https://doi.org/10.1093/bja/aei210
Baldo, B. A., & Rose, M. A. (2020). The anaesthetist, opioid analgesic drugs, and serotonin toxicity: a mechanistic and clinical review. British Journal of Anaesthesia, 124(1), 44–62. https://doi.org/10.1016/j.bja.2019.08.010
Thomas, S. J., Shin, M., McInnis, M. G., & Bostwick, J. R. (2015). Combination therapy with monoamine oxidase inhibitors and other antidepressants or stimulants: strategies for the management of treatment-resistant depression. Pharmacotherapy, 35(4), 433–449. https://doi.org/10.1002/phar.1576

This source was quoted in other studies, should be good to use? 
Sansone, R. A., & Sansone, L. A. (2009). Tramadol. Psychiatry (Edgmont), 6(4), 17–21. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2714818/

---
Before expanding into opioids, here are my thoughts. I will be making a post after this discussing the classification issues, namely categories of drugs. If this is accepted and moved forward, then instead of "opioids" we will replace it with an opioid such as oxycodone or heroin. Amphetamines could be replaced with amphetamine, etc. This would remove interactions such as Opioids + MAOI's where some are unsafe and some are low risk. I would propose doing a separate interaction for methadone, as the ss concern would likely not apply for what it is changed over to.

**Reply by utaninja** [reply]
> Before expanding into opioids, here are my thoughts. I will be making a post after this discussing the classification issues, namely categories of drugs. If this is accepted and moved forward, then instead of "opioids" we will replace it with an opioid such as oxycodone or heroin. Amphetamines could be replaced with amphetamine, etc. This would remove interactions such as Opioids + MAOI's where some are unsafe and some are low risk. I would propose doing a separate interaction for methadone, as the ss concern would likely not apply for what it is changed over to.

As sent on the issue just made- We've discussed this in a few places. Ultimately the ideal goal is going to be we have interactions specific to drugs, not categories, and update the relevant interactions for stuff like this and amphetamines where there are some serotonin syndrome concerns (etc) depending on the drug within the class.

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/331
